data/transactions/logic/assembler_test.go

```diff
@@ -1446,16 +1396,11 @@ int 1
 	require.NoError(t, err)
 	require.Equal(t, ops1.Program, ops.Program)
 
-	_, err = AssembleStringWithVersion(text, 0)
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "version mismatch")
-
-	_, err = AssembleStringWithVersion(text, 2)
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "version mismatch")
+	testProg(t, text, 0, expect{1, "version mismatch..."})
+	testProg(t, text, 2, expect{1, "version mismatch..."})
+	testProg(t, text, assemblerNoVersion)
 
 	ops, err = AssembleStringWithVersion(text, assemblerNoVersion)
-	require.NoError(t, err)
```

Comment:
no error check makes sense, otherwise ops is nil and sigfault instead of nice test failure message

Reply:
Agreed, not sure how that got removed.  Thanks.